pyaerocom/aeroval/collections.py

```diff
@@ -87,13 +87,18 @@ def keylist(self, name_or_pattern: str = None) -> list[str]:
         KeyError
             if no matches can be found
         """
+        # Special case where the model cfg is empty, used for obs-only
+        if self._entries.keys() == []:
```

Comment:
Shouldn't this be on `ModelCollection`? Now it applies to both `ObsCollection` and `ModelCollection` since it's on the base class.

Reply:
Good observation, moved to ModelCollection